_toc.yml

```diff
@@ -132,6 +132,8 @@ entries:
           entries:
             - file: docs/tools/terraform/howto/upgrade-provider-v1-v2
               title: Upgrade the Aiven Terraform Provider from v1 to v2
+            - file: docs/tools/terraform/howto/config-postgresql-provider
+              title: Configure PostgreSQL provider for Aiven PG services  
```

Comment:
Proposed title: Use PostgreSQL provider alongside Aiven Terraform Provider

Reply:
Hi @dewan-ahmed I have make the changes you have suggested and pushed them. I will resolve all the conversations.